Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +263,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    final protected function getQuestionHelper()
```

Comment:
Yes, this should either `private` or `final protected` if there is a reason. 

This method did only exist for BC reasons and will be removed when dropping old symfony version. Am I right? So please make this `private`